packages/@aws-cdk/toolkit-lib/lib/api/garbage-collection/garbage-collector.ts

```diff
@@ -178,6 +178,20 @@ interface GarbageCollectorProps {
    * @default true
    */
   readonly confirm?: boolean;
+
+  /**
+   * Stack names to ignore during garbage collection
+   *
+   * @default []
+   */
+  readonly ignoreNonCdkStacks?: string[];
```

Comment:
The way this is implemented we are skipping _any_ stack, not just CDK stacks. Like @kaizencc I'm not convinced this is a good idea in the first place, but at leat the option need to be named correctly.